### PR TITLE
Added landing page

### PR DESCRIPTION
**Pull request by jeetsdev**
## Added landing page and created a folder structure

- Landing area
- Shop by category 
- New arrivals

[Preview](https://deploy-preview-1--thriving-gumption-494d22.netlify.app/)

### CODE REVIEW

**Review by AnjaliDeshwani — APPROVED** [review]
Great Folder Structure. Changes looks good to me.

File: src/assets/index.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import TeesTwo from "./tees/tees-2.png";
```
Comment (xsaurabhsuthar) [inline src/assets/index.js:1]: you can use barrel export for importing images.

File: src/components/Header/Header.jsx [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+                {/* Search section here*/}
+                <div className="nav__search-sec display__none">
+                    <form action="" className="center__flex">
+                        <i className="fas fa-search margin-8px txt-grey" />
+                        <input type="text" placeholder="Search items here" />
+                    </form>
+                </div>
+                {/* Button section here*/}
+                <div className="nav__button-sec center__flex">
+                    <a
```
Comment (xsaurabhsuthar) [inline src/components/Header/Header.jsx]: You can replace a tag with Link(react-router-dom).

**Review by xsaurabhsuthar — APPROVED** [review]
Looks good to merge 🚀🚀🚀🚀

**Pull request merged**